test/e2e_node/runner/remote/run_remote.go

```diff
@@ -468,26 +472,30 @@ func (a byCreationTime) Less(i, j int) bool { return a[i].creationTime.After(a[j
 func (a byCreationTime) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
 // Returns a list of image names based on regex and number of previous images requested.
-func getGCEImages(imageRegex, project string, previousImages int) ([]string, error) {
+func getGCEImages(imageRegex, imageFamily string, project string, previousImages int) ([]string, error) {
 	ilc, err := computeService.Images.List(project).Do()
```

Comment:
There's a limit on the number of images that can be returned from this call; we should use the Pages() method instead but that's outside the scope of this change.

Reply:
Does this call exclude global images?

Reply:
```
// List: Retrieves the list of custom images available to the specified
// project. Custom images are images you create that belong to your
// project. This method does not get any images that belong to other
// projects, including publicly-available images, like Debian 8. If you
// want to get a list of publicly-available images, use this method to
// make a request to the respective image project, such as debian-cloud
// or windows-cloud.
```

Reply:
> There's a limit on the number of images that can be returned from this call; we should use the Pages() method instead but that's outside the scope of this change.

Done